lib/modules/versioning/bazel-module/bzlmod-version.spec.ts

```diff
@@ -0,0 +1,293 @@
+import { BzlmodVersion, Identifier, VersionPart } from './bzlmod-version';
+
+describe('modules/versioning/bazel-module/bzlmod-version', () => {
+  describe('Identifier', () => {
+    it('when all digits', () => {
+      const ident = new Identifier('123');
+      expect(ident.asString).toBe('123');
+      expect(ident.asNumber).toBe(123);
+      expect(ident.isDigitsOnly).toBe(true);
+    });
+
+    it('when not all digits', () => {
+      const ident = new Identifier('foo123');
+      expect(ident.asString).toBe('foo123');
+      expect(ident.asNumber).toBe(0);
+      expect(ident.isDigitsOnly).toBe(false);
+    });
+
+    it.each([
+      { a: '1', b: '1', exp: true },
+      { a: '1', b: '2', exp: false },
+      { a: 'foo1', b: '1', exp: false },
+      { a: 'a', b: 'a', exp: true },
+      { a: 'a', b: 'b', exp: false },
```

Comment:
Please, use template literal syntax. Search our codebase for:
```
each`
```

Reply:
Sorry. I missed that bullet point in the best-practices.